feature_engine/mathematical_combination.py

```diff
@@ -0,0 +1,94 @@
+from feature_engine.base_transformers import BaseNumericalTransformer
+
+
+class MathematicalVariableCombinator(BaseNumericalTransformer):
+    """
+    The MathematicalVariableCombinator() applies basic mathematical operations across features,
+    returning one additional feature as a result.
+
+    Parameters
+    ----------
+
+    variables: list, default=None
+        The list of numerical variables to be transformed. If None, the transformer
+        will find and select all numerical variables.
+
+    math_operations: list, default=['sum', 'prod', 'mean', 'std', 'max', 'min']
+        The list of basic math operations to be used in transformation.
+
+        Each operation should be a string and must be one of elements
+        from list: ['sum', 'prod', 'mean', 'std', 'max', 'min']
+
+        Each operation will result in operation column in result dataset.
+
+    variables_set_alias: string, default=None
+        Alias for variables set, allow for replacing all columns name from result column with this string.
```

Comment:
Name of the newly created variables. The user can enter a name or a list of names for the newly created features (recommended). User must enter one name for each mathematical transformation indicated in the math_operations attribute. That is, if you want to perform mean and sum of features, you should enter 2 new variable names. If you perform only mean of features, enter 1 variable name. Alternatively,  if you chose to perform all mathematical transformations, please enter 6 new variable names. 

The name of the variables indicated by the user should coincide with the order in which the mathematical operations are initialised in the transformer. That is, if you set math_operations = ['mean', 'prod'], the first new variable name will be assigned to the mean of the variables and the second variable name to the product of the variables.

If new_variable_names=None, the transformer will assign an arbitrary name to the newly created features starting by the name of the mathematical operation, followed by the variables combined separated by -. 

Could you add the above to the docstrings?

Reply:
thx, I'll change this in next commit